internal/tests/vm_internal_disk_resource_test.go

```diff
@@ -0,0 +1,103 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+//go:generate go run github.com/FrangipaneTeam/tf-doc-extractor@latest -filename $GOFILE -example-dir ../../examples -test
+const testAccVMInternalDiskResourceConfig = `
+resource "cloudavenue_vm_internal_disk" "example" {
+  vapp_name       = "vapp_test3"
+  vm_name         = "TestRomain"
```

Comment:
Please change `TestRomain` to more generic

Reply:
yes... waiting vapp/vm resources to do this more properly.